pandas/tests/io/test_pytables.py

```diff
@@ -5,6 +5,7 @@
 from io import BytesIO
 import os
 import tempfile
+import time
```

Comment:
what are you using this for?

Reply:
Used this for dynamically generating unix time stamps (`time.time()`) before writing them to the HDF5 file, but made this static now.